pkg/monitor/backenddisruption/disruption_backend_sampler.go

```diff
@@ -525,6 +545,8 @@ func (b *disruptionSampler) consumeSamples(ctx context.Context, interval time.Du
 	// when we exit this function, we want to set a final duration of failure.  We don't actually know whether it ended
 	// or how long it took to ask
 	defer func() {
+		fmt.Fprintf(os.Stderr, "%v setting the intervals as done at time %v\n", b.backendSampler.locator, time.Now())
```

Comment:
Why is this stderr? 

Would prefer we use logrus whenever possible, consistent structured logging would be a win for us, we're often staring at unstamped log messages wondering what happened when and how long something took.